### DISCUSSION TITLE
Can I access my books on the go?

### DISCUSSION THREAD

**Discussion by Drakemoor** [advplyr/audiobookshelf#2536] [Q&A] [answered]
I just installed the new ABS windows https://github.com/mikiher/audiobookshelf-windows 

But I can find a way to access my library when I'm away from home. I'm using the Android version to access the library, and it works well when I'm home, but I see no option to make it accessible from outside. 

Can someone point me to a way to do that? 

Thanks.

**Comment by ezzfool** [comment]
I recently got Tailscale up and running with Audiobookshelf on a Docker install in Ubuntu Server, and then even managed to set it up on my Android client. I wonder if this might work for you on a Windows server environment.

**Comment by portrman** [comment]
I'd suggest VPN from your phone into your house.  My gear supports wireguard on the server side and there's an android client for wireguard.  If your network gear doesn't natively support it, you can google 'wireguard server windows' and then poke some holes to run it that way.  I run a ubiquiti dream machine pro at home but I'm guessing other router/fw/all-in-ones support wireguard or some other vpn solution as well.

**Comment by DavidPesticcio** [comment]
You could use a Cloudflare "tunnel"

- Use a domain held by Cloudflare, either an existing domain, a new domain, or transfer one
- Create a tunnel with a public URL from above domain, with the "cloudflared" cli or Cloudflare dashboard
- Use the "cloudflared" tool to activate the tunnel from your private host to Cloudflare
- Use the public URL of the domain used in the second step to access your private Audiobookshelf server

More info: https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/

As a temporary fix to "try it out" you can use Cloudflare "Quick Tunnels" with none of the above overheads.

Literally run this on your private Audiobookshelf server to make it available via a public URL - **be sure you have a password on your root user!!!!**

```
cloudflared tunnel --url http://localhost/
```

Ensure you use the port that Audiobookshelf is listening on! - the default is `80`

More info: https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/do-more-with-tunnels/trycloudflare/

HTH

**Comment by brianalsum** [comment, marked as answer]
Tail Scale is the easiest to get up and running. works really well.

**Reply by Drakemoor** [reply]
Thank you! This looks promising.
